src/script/index.js

```
@@ -1 +1,132 @@
 import '../styles/main.scss';
+let hamIcon = document.getElementById('hamIcon');
+let crossIcon = document.getElementById('crossIcon');
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'crossIcon' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)